### PR TITLE
score

### PR DESCRIPTION
**Pull request by srossross**
initial health risk score

### CODE REVIEW

File: score/score/health_risk.py
Comment (karamba228) [inline score/score/health_risk.py:1]: This scoring only accounts for Github statistics.
I think we should also add:
- License check (with less permissive license a person using the package for proprietary projects may run into legal issues therefore further investigation needed or "caution needed")
- Maintainer status (if the maintainer is absent the package may be out of date with current security standards)
- Last update (if the project's last update was longer than a year ago, it is most likely out of date and needs to be marked as caution needed or even high risk)

File: score/score/health_risk.py [hunk 1]
```diff
@@ -1,3 +1,11 @@
+SCORE_ORDER = ["Healthy", "Caution Needed", "Moderate Risk", "High Risk"]
+
+HEALTHY = SCORE_ORDER.index("Healthy")
+CAUTION_NEEDED = SCORE_ORDER.index("Caution Needed")
+HIGH_RISK = SCORE_ORDER.index("Moderate Risk")
+HIGH_RISK = SCORE_ORDER.index("High Risk")
```
Comment (karamba228) [inline score/score/health_risk.py:6]: should HIGH_RISK be duplicated?

Comment (srossross) [reply score/score/health_risk.py:6]: you are right, fixed

File: score/score/health_risk.py
Comment (srossross) [reply score/score/health_risk.py:1]: * #83
* #84 

> maintainer is absent

what does this mean?

Comment (karamba228) [reply score/score/health_risk.py:1]: We are getting 2 fields from pypi: `maintainer `and `maintainer_email`, I thought they could be useful. But looking over the data, all projects return empty string for these fields. So we can skip that, I'll create a ticket for cleaning up data fields that always return null or empty

**Review by karamba228 — APPROVED** [review]
Line 13: score["value"] = "Unkonwn" 

There is a spelling error, other than that, looks good

**Pull request merged**